### PR TITLE
Selecting the right Settings section

### PR DESCRIPTION
**Pull request by ShimonLeb**
Closes rubberduck-vba/Rubberduck#4610 and 2 more settings icon with the same problem.

I`m new here (and on GitHub) so please let me know if I did it correctly

### CODE REVIEW

**Comment by retailcoder** [comment]
You're doing great! Question: would you happen to have a Stack Overflow / Stack Exchange account? You can join our dev chat [here](https://chat.stackexchange.com/rooms/14929/vba-rubberducking) (a moderator will be able to grant you write access if you don't have the required 20 rep points).

**Comment by ShimonLeb** [comment]
@retailcoder  Thanks. how do I request write access?

File: Rubberduck.Core/UI/Settings/SettingsForm.cs [hunk 1]
```diff
@@ -12,6 +12,7 @@ namespace Rubberduck.UI.Settings
     public interface ISettingsFormFactory
     {
         SettingsForm Create();
+        SettingsForm Create(SettingsViews activeView);
```
Comment (bclothier) [inline Rubberduck.Core/UI/Settings/SettingsForm.cs]: I'm probably being oblivious -- where's the implementation?

Comment (ShimonLeb) [reply Rubberduck.Core/UI/Settings/SettingsForm.cs]: I thought (maybe I`m wrong, but I tested it and it works) that the IoC is passing the `activeView ` into the `SettingsForm` constructor where it`s passed to the `SettingsControlViewModel` constructor where it is set here `SelectedSettingsView = SettingsViews.First(v => v.View == activeView);
`

**Pull request merged**